packages/flex-dev-utils/src/fs.ts

```diff
@@ -45,16 +46,30 @@ export const checkFilesExist = (...files: string[]) => {
  */
 export const getPackageJsonPath = (forModule: boolean = false) => path.join(process.cwd(), 'package.json');
 
+/**
+ * Gets package-lock.json path
+ */
+export const getPackageLockJsonPath = (forModule: boolean = false) => path.join(process.cwd(), 'package-lock.json');
+
 /**
  * Updates the package.json version field
  *
  * @param version the new version
  */
 export const updatePackageVersion = (version: string) => {
+  const indent = detectIndent(fs.readFileSync(getPackageJsonPath(), 'utf8')).indent;
+
   const packageJson = readPackageJson();
   packageJson.version = version;
 
-  fs.writeFileSync(getPackageJsonPath(), JSON.stringify(packageJson, null, 2));
+  fs.writeFileSync(getPackageJsonPath(), JSON.stringify(packageJson, null, indent));
```

Comment:
While I understand this use case, we do want to impose good coding practice and JSON coding practice is 2 indentation; so let's keep as two.

Reply:
OK, I'll remove this.

But IMO a third party package should not reformat files to its taste, it should respect the author's style. I understand that 2 spaces is common, but it's not any better that 4 spaces or 8.